ObjectiveGit/GTRepository.h

```diff
@@ -565,6 +555,16 @@ NS_ASSUME_NONNULL_BEGIN
 /// Returns whether `ahead` and `behind` were successfully calculated.
 - (BOOL)calculateAhead:(size_t *)ahead behind:(size_t *)behind ofOID:(GTOID *)headOID relativeToOID:(GTOID *)baseOID error:(NSError **)error;
 
+/// Creates an enumerator for walking the local commits beginning at one OID and
+/// ending at another. It will *not* include the commit for `endingOID`.
+///
+/// startingOID - The starting OID.
+/// endingOID   - The ending OID.
+/// error       - The error if one occurred.
+///
+/// Returns the enumerator or nil if an error occurred.
+- (nullable GTEnumerator *)enumeratorForCommitsStartingAtOID:(GTOID *)startingOID endingAtOID:(GTOID *)endingOID error:(NSError **)error;
```

Comment:
I still don't love this name—I think `endingAt` in particular is a little misleading. :grimacing:

How about `enumeratorForUniqueCommitsFromOID:relativeToOID:` or `enumeratorForCommitsFromOID:excludingCommitsFromOID:`?